docs/apis/streaming/connectors/redis.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Redis Connector"
+
+# Sub-level navigation
+sub-nav-group: streaming
+sub-nav-parent: connectors
+sub-nav-pos: 4
```

Comment:
rabbitmq.md has also sub-nav-pos 4 (I guess this will result in an conflict). did you build the documentation and checked it?